qa/suites/system-tests/object/multi-site/clusters/single_realm_cluster.yaml

```diff
@@ -0,0 +1,53 @@
+overrides:
```

Comment:
I think moving this file to https://github.com/red-hat-storage/ceph/tree/master/qa/suites/system-tests/clusters makes more sense and create a symlink for it multisite dir.

Reply:
updated.